src/main/groovy/org/shipkit/internal/gradle/ReleaseNotesPlugin.java

```diff
@@ -19,9 +19,7 @@
  *
  * <ul>
  *     <li>fetchReleaseNotes - fetches release notes data, see {@link ReleaseNotesFetcherTask}</li>
- *     <li>updateReleaseNotes - updates release notes file in place, see {@link IncrementalReleaseNotes.UpdateTask}</li>
- *     <li>previewReleaseNotes - prints incremental release notes to the console for preview,
- *          see {@link IncrementalReleaseNotes.PreviewTask}</li>
+ *     <li>updateReleaseNotes - updates release notes file in place, or only displays preview, see {@link UpdateReleaseNotesTask}</li>
```

Comment:
How about we document the project property here so that we have comprehensive docs.